app/metrics.yaml

```diff
@@ -97,7 +97,7 @@ tabs:
     bugs:
       - https://github.com/MozillaReality/FirefoxReality/issues/1609
     data_reviews:
-      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
```

Comment:
The URL is not the request one, it would be the URL of Data Review Form.

Reply:
Fixed